docs/application/dotnet/guides/applications/uiapplication/ui-app.md

```diff
@@ -13,10 +15,10 @@ To create a basic UI application, you must:
 
 - Manage [application states and transitions](#state_trans) during the
     application life-cycle.
+
 - Define a [background category](#allow_bg) for your application, if
     you want it to run in the background.
 
-
 <a name="callback"></a>
 ## Event Handling
```

Comment:
2nd headings should use sentence case capitalization. Also, this seems to be a case of a gerund. Perhaps a more appropriate heading would be "**_Handle events_**"
@mijin85cho can you please confirm this assessment?
```suggestion
## Event handling
```